libs/test-common/src/main/java/org/codice/ddf/test/common/features/FeatureUtilities.java

```diff
@@ -196,7 +151,31 @@ public static void uninstallFeature(FeaturesService featuresService, String feat
           .put(Bundle.ACTIVE, "ACTIVE")
           .build();
 
-  private static void printInactiveBundles(
+  public void printInactiveBundles() {
```

Comment:
❓ Can we make these and the following methods private? Especially the service related ones, I don't anyone should be using a different bundle context to load services.

What are your thoughts on creating an interface to prevent exposing methods such as these since we're now using it as a service?

Reply:
I thought creating an interface would be overkill for a test utility. To add onto that, I was under the impression that this would all change when `waitForReady` was redone soon.